src/main/java/com/amihaiemil/docker/RtContainer.java

```diff
@@ -55,8 +55,8 @@ public JsonObject inspect() throws IOException {
             info = Json
                 .createReader(response.getEntity().getContent()).readObject();
         } else {
-            throw new IllegalStateException(
-                "Container#inspect() expected status 200, but got " + status
+            throw new UnexpectedResponseException(
+                "Containers#inspect()", status, HttpStatus.SC_OK
```

Comment:
@amihaiemil the first argument to ctor of `UnexpectedResponseException` is supposed to be the endpoint